invenio_accounts/profiles/schemas.py

```diff
@@ -21,6 +21,14 @@ def validate_visibility(value):
         )
 
 
+def validate_preferences_locale(value):
+    """Check if the value is a valid locale."""
+    if False:  # TODO upddate this
```

Comment:
we need to find a reliable way to validate a locale string, from a quick check it is not straightforward :)